src/js/translations/en-US.json

```diff
@@ -0,0 +1,10 @@
+{
+  "DASHBOARD.VIEW_ALL": "View all {servicesCount} Services",
```

Comment:
Let's think about how we could handle the pluralization. i.e. 1 Service or X Services